app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/notes/NotesViewModel.kt

```diff
@@ -101,6 +101,19 @@ class NotesViewModel @Inject constructor(
         )
     }
 
+    fun deleteQuestion(notesIndex:Int, questionIndex: Int) {
```

Comment:
I am now working on remove the dependency of question list from Note data class and use Question data class, do you want to just keep this and let me do further changes once I have done the Question entity or modify this once I have finished the data class changes?

Reply:
I am removing the code of delete but keeping the empty function. I can add the logic when Data changes are done. So right now clicking delete from UI will not do anything

Reply:
Hello @sev3rus 
You can continue implementing the database and other classes related to Questions. The deleteQuestion function in NotesViewModel is not dependent on your implementation. Once you create the CRUD methods for Questions then if required I can modify the method.